components/action-mgt/org.wso2.carbon.identity.action.management/src/test/java/org/wso2/carbon/identity/action/management/dao/ActionManagementDAOImplTest.java

```diff
@@ -108,8 +110,17 @@ public void wrapUp() throws Exception {
         closeH2Database();
     }
 
-    @Test(priority = 1)
-    public void testAddAction() throws ActionMgtException {
+    @DataProvider(name = "actionProvider")
+    public Object[][] actionDataProvider() {
+
+        return new Object[][]{
+                {Action.ActionTypes.PRE_ISSUE_ACCESS_TOKEN.getActionType(), preIssueAccessTokenAction},
+                {Action.ActionTypes.AUTHENTICATION.getActionType(), authenticationAction}
+        };
+    }
```

Comment:
Do we really have a difference on onboarding this data provider when it comes to test execution ?